src/main/java/org/terasology/signalling/components/ScrewdriverComponent.java

```diff
@@ -17,5 +17,8 @@
 
 import org.terasology.entitySystem.Component;
 
+/**
+ * A component that indicates its parent {@link EntityRef} is a screwdriver item.
```

Comment:
Nitpick: I'm not sure components are really parented to EntityRefs. They are more attached to one. Parentage could perhaps be confused in the rare cases when a Component is based on something more specific, or other parent related stuff

So I'd go "... indicates the EntityRef it is attached to is a .."

Reply:
Fixed, I used `"parent"` to keep it short, not sure if there is a name for the thing a component is attached to.